card/Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'http://rubygems.org'
+
+gemspec
+
+#gem 'card', :path=>'.'
```

Comment:
I guess that is typical for gems, they can add tasks, but you don't usually run anything from the gem dir.